src/axom/sidre/core/Buffer.cpp

```diff
@@ -299,8 +302,14 @@ void Buffer::importFrom(conduit::Node& buffer_holder)
   {
     allocate();
     conduit::Node& buffer_data_holder = buffer_holder["data"];
-    copyBytesIntoBuffer(buffer_data_holder.element_ptr(0),
-                        buffer_data_holder.total_strided_bytes());
+
+    const auto num_bytes = buffer_data_holder.total_strided_bytes();
+    if(num_bytes > 0)
```

Comment:
This checks that we're only attempting to copy bytes into a buffer when there are actually bytes to copy.
(Note: `copyBytesIntoBuffer` was already doing the right thing in this case, so this will not change behavior. The code was previously issuing a warning in this case though).